sdk/storage/storage-blob/samples/storage-blob-resize-function/Thumbnail/index.js

```diff
@@ -0,0 +1,44 @@
+const Jimp = require('jimp');
```

Comment:
The file only export a module, but no one to invoke it. It'd be better to make it directly runnable.

Reply:
It's invoke by the function `Jimp.read()` in the row sixteen.